src/applications/login/utilities/downtimeBanners.js

```diff
@@ -0,0 +1,44 @@
+import { EXTERNAL_SERVICES } from 'platform/monitoring/external-services/config';
```

Comment:
this felt like some added bloat in the sign-in component, so I pulled it into a array of configs to just be pulled in and rendered